src/client/components/network-editor/index.js

```diff
@@ -93,17 +93,40 @@ export class NetworkEditor extends Component {
     ]);
 
     const enableSync = async () => {
+      console.log('Starting to enable sync in editor');
+
       try {
+        console.log('Loading');
```

Comment:
Should we keep these log statements?

Reply:
I think that they're useful for troubleshooting for now, but they're not strictly necessary